docs/creating_component_code_review.md

```diff
@@ -47,8 +49,15 @@ bridge = Bridge(...)
 status = bridge.status()
 ```
 
-### 5. Limit platforms included in initial pull request
-Large pull requests mean there is a larger chance of finding problems that need to be addressed, and more code that needs to be reviewed between every requested change. If your new component will have multiple platforms, try and limit your initial pull request to a single platform. Once the initial component is merged, you can submit additional PRs for the remaining platforms. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new feature in sooner. Pull requests containing large code dumps will not be a priority for review.
+### 5. Limit initial pull request to Minimum Viable Product (MVP)
+Large pull requests mean there is a larger chance of finding problems that need to be addressed, and more code that needs to be reviewed between every requested change. Limit the initial pull request to the minimum functionality needed for someone to get value out of the integration. Once the initial pull request is merged, you can submit additional PRs for the remaining features. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new integration/features in sooner. **Pull requests containing large code dumps (> 800 line adds excluding tests) will not be a priority for review and may be closed.** Limit MVPs to the following:
```

Comment:
There is also a greater risk of missing issues and if the developer is inexperienced a bad design.
Should we really explicitly have a reference to lines of codes? The inclusion of tests (which is required in some cases) also greatly increases the amount of code

Reply:
This item is way too much prose for a check _list_. The message gets lost.

```suggestion
### 5. Make your pull request as small as possible

Do not solve several problems in a single pull request.

Do not mix clean-ups and new features in a single pull request.

Limit a new integration to the minimum functionality needed for someone to get value out of the integration. For example, only add a single platform in the initial pull request.

Do not submit pull requests that depend on other work which is still unmerged.
```

Reply:
He explicitly excludes tests from that count. But I tend to agree, a MVP should be a short as possible.

Reply:
Shorter is always better.
- I do think there is a place for minimal explanation as that helps our community understand _why_.
- Note that this page is titled "checklist for _creating_ components".  Some of your suggestions are general PR guidelines (which I completely agree with).  This page probably gets used for that too, but think we should focus on creation.
- I think we should quantify what constitutes "large" (too large).  We do for small (it's 30 additions).

Reply:
I agree that only my third item relates specifically to components.

We have (at least) three checklists with some duplication. Maybe they should be merged to one, with small sections for specific component/platform advice.

Reply:
I think that's a great idea and would be happy to help on that in a separate PR because I definitely that that pain when I first started contributing.